data/comid/README.md

```diff
@@ -0,0 +1,712 @@
+| lang         	| String 	| Language/country code.                                  	| "en-GB"                                      	|   	
+| tag-identity 	| Object 	| Identity of this CoMID tag (UUID + version).            	| "id": "43BBE37F-2E61-4B33-AED3-53CFF1428B16" 	|   	
+| entities     	| Array  	| The organizations/roles associated with this CoMID tag. 	| [ { "name": "ACME Ltd." ... } ]              	|   	
+
+###  Reference-Value Fields
+|        Field       |  Type  |                                     Description                                    |                                            Example                                            |
+|:------------------:|:------:|:----------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------:|
+| environment        | Object | Contains class and optionally instance, layer, index.                              | See 3.1 Environment.                                                                          |
+| measurements       | Array  | List of measurement objects.                                                       | [ { "key": { ... }, "value": { ... } } ]                                                      |
+| measurements.key   | Object | Identifies the measurement. Could be a psa.refval-id, cca.platform-config-id, etc. | { "type": "psa.refval-id", "value": { "label": "BL", "version": "2.1.0", ... } }              |
+| measurements.value | Object | Holds the actual measurement data.                                                 | { "digests": ["sha-256:..."] }, or { "raw-value": { "type": "bytes", "value": "..." } }, etc. |
+
+###  Attester-Verification-Key Fields
+|       Field       |  Type  |               Description               |                                   Example                                   |
+|:-----------------:|:------:|:---------------------------------------:|:---------------------------------------------------------------------------:|
+| environment       | Object | Defines the environment for these keys. | See 3.1 Environment.                                                        |
+| verification-keys | Array  | Holds one or more public keys.          | [ { "type": "pkix-base64-key", "value": "-----BEGIN PUBLIC KEY-----..." } ] |
+----------
+
+###  High Level Structure for CoMID Templates
+
+```mermaid
+graph TD
+    A[CoMID Template] --> B[Global Fields]
+    A --> C[Triples]
+    B --> B1[lang]
+    B --> B2[tag-identity]
+    B --> B3[entities]
+    C --> C1[Reference Values]
+    C --> C2[Attester Verification Keys]
+    C1 --> C1a[Environment]
+    C1 --> C1b[Measurements]
+    C2 --> C2a[Environment]
+    C2 --> C2b[Verification Keys]
+```
+
+##  Full Examples and Walkthroughs
+
+Below are the **seven** template files, each highlighting different aspects of CoMID usage.
+
+###  comid-cca-mult-refval.json
```

Comment:
(I thought I had made this comment in the previous review cycle, but I cannot find it here, so...)

Rather than making verbatim copies of the files (which will eventually go out of sync), it's better to reference the originals here.

GitHub has a "code snippet" functionality which comes in handy: by copying the permalink to the JSON file (and adding the wanted lines range), the file is embedded in the rendered markdown, which is pretty cool.

Reply:
To be a bit more concrete: using the permalink to `comid-cca-mult-refval.json` (including explicit line range) `https://github.com/veraison/cocli/blob/0d8fae8210ae527589792de2fba54442302380f7/data/comid/templates/comid-cca-mult-refval.json#L1-L93` will render as:

https://github.com/veraison/cocli/blob/0d8fae8210ae527589792de2fba54442302380f7/data/comid/templates/comid-cca-mult-refval.json#L1-L93